dev-system/project.clj

```diff
@@ -49,7 +49,18 @@
   :url "https://github.com/nasa/Common-Metadata-Repository/tree/master/dev-system"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies ~(concat '[[org.clojure/clojure "1.8.0"]
+  :dependencies ~(concat '[;; XXX REMOVE the following dep when the stubbed
```

Comment:
I don't think we have to put these in dev-system since it already requires the search-app which will pull in the dependencies.

Reply:
Cool; I'll take them out (I needed them for testing in the REPL).